### ISSUE TITLE
Taxi2 does not function with fasta file

### ISSUE THREAD

**Issue by mvences** [iTaxoTools/TaxI2-legacy#37]
So far, I have always tried Taxi2 with a tabfile as input, which worked very well. Also the Genbank file I tried today performed well. However, when I tried a fasta file, the program crashed (with large input files) or just stopped without output (with small input files). I think this is because it tries to calculate values such as distances between species and similar, which does not work of course because the information is missing. I could send you a screen shot of the error messages, but it will be easier if you just try yourself with a small fasta input file to see where the problem is. 

Basically, when input is specified as fasta, everything related to calculation of distances between species or genera should just be dropped, no figures need to be outputted, and of course also in the cluster output all the additional information regarding species should not be calculated.
